modules/owesome.js

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-sequences */
+/* eslint-disable no-plusplus */
+/* eslint-disable indent */
+/* eslint-disable class-methods-use-this */
+/* eslint-disable eqeqeq */
+/* eslint-disable consistent-return */
+/* eslint-disable array-callback-return */
+/* eslint-disable no-unused-expressions */
+export default class Books {
+  constructor() {
+    this.BookList = {};
```

Comment:
- [ ] It would be nice if you could fix these linter checks so your code is professional and standard. Disabling the linter checks defies its purpose and it's there to help make you write a clean, modern, professional, and error-free codebase. Kindly fix them 👍